gips/0080-optimistic-ebo.md

```diff
@@ -0,0 +1,237 @@
+    - Other bridges require case-by-case analysis.
+
+> Using an Arbitrator (a Council of qualified individuals selected by the governance) is the simplest way to launch the system at this early stage and manage all the chains with a single resolution mechanism. This approach allows for greater decentralization in the future, as the source of truth can be easily migrated.
+> 
+
+**Some considerations:**
+
+- Implementing multiple resolution systems by chain or type (rollups, L1s, etc) adds complexity.
+- The Council should upgrade the system to accommodate future improvements.
+- Disputes should be managed per-chain to prevent a single dispute from impacting the entire system.
+
+## **Economic Security considerations**
+
+We recognize that economic security in The Graph presents complex challenges, which various teams are researching. The ideas discussed here are preliminary and may adapt to future protocol developments.
+
+### Parameters
+
+We suggest the following parameter values for the Escalation Bond system:
+
+- **Bond threshold**: The amount of stake that triggers a call to the Arbitrator (virtual bond in the escalation system), set at 100K GRT.
+- **Round**: The maximum number of rounds a dispute can escalate, set at 5.
+- **Min Bond**: Minimum amount required for five rounds of escalations to reach the Bond threshold, set at 11111 GRT.
+- **Dispute Window**: An indexer's maximum response time to a dispute is 30 minutes.
+
+These values are based on an economic security analysis that evaluates the potential gains from corruption across various attack scenarios. You can read the complete analysis [here](https://www.notion.so/Economic-security-considerations-1009a4c092c780b6afb1fa8bb8c266be?pvs=21).
+
+**The Arbitrator has the authority to modify these parameters.**
+
+# **Detailed Specification**
+
+## Prophet modules
+
+Interaction with Prophet requires the following modules that we will use/build to support the EBO:
+
+- **RequestModule:** Used to request data in prophet. We will build the EBORequestModule which will allow users to create epoch-block requests to the oracle with the needed data for it. Compatible with partial requests (a subset of all the chains) and will contain the epoch number and the chain to fetch for that epoch.
+- **ResponseModule:** Used by data providers to answer requests.
+- **DisputeModule:** Provides the framework to automatically solve disputes before going into the resolution module. The [BondEscalationModule](https://github.com/defi-wonderland/prophet-modules/blob/dev/solidity/contracts/modules/dispute/BondEscalationModule.sol) manages disputes by allowing users to bond tokens to vote for the user that is telling the truth. The only modification we need to add is that only one propose per request can be solved through escalation. We can safely remove this.
```

Comment:
I am not clear if you are suggesting we need or that we do not need more than one.
```suggestion
- **DisputeModule:** Provides the framework to automatically solve disputes before going into the resolution module. The [BondEscalationModule](https://github.com/defi-wonderland/prophet-modules/blob/dev/solidity/contracts/modules/dispute/BondEscalationModule.sol) manages disputes by allowing users to bond tokens to vote for the user that is telling the truth. The only modification we need to add is that only one proposal per request can be solved through escalation. We can safely remove this.
```